h2o-algos/src/main/java/hex/tree/isoforextended/ExtendedIsolationForestModel.java

```diff
@@ -119,6 +119,18 @@ public long progressUnits() {
          */
         public int _score_tree_interval = 0;
 
+        /**
+         * For _initial_score_interval milliseconds - score each iteration of the algorithm.
+         */
+        public int _initial_score_interval = 4000;
```

Comment:
Is it possible to set these values somehow? In the API PR (https://github.com/h2oai/h2o-3/pull/6038) I can't see it. Or is this default setting if the manual scoring is not set, so it cannot be changed?

Reply:
No didn't found it in an APIs of other algos. It's possibile to set in only in backend with parms._initial_score_interval..

Reply:
I think there could be a possibility to set this parameter, what about using system property to set up it? WDYT @valenad1?

Reply:
I don't think so, we could make it public, when someone ask for it 👍